### DISCUSSION TITLE
Maps.app

### DISCUSSION THREAD

**Discussion by alexmyczko** [trunkmaster/nextspace#389] [Ideas]
there exists KDE marble, GNOME maps, but nothing for gnustep. or maybe once there's a webkit based gnustep webbrowser. i guess you're too busy to quickly write a gnustep-maps.app ? ;)

**Comment by trunkmaster** [comment]
I have to finish Preferences, Workspace, Terminal and Login to have a release. So, yes, I'm quite busy at the moment... Do you? ;)
Maybe we need a wishlist for applications with short description of desired functonality? Somebody will implement it someday... ;)

**Comment by alexmyczko** [comment]
I think there's already one here: http://wiki.gnustep.org/index.php/Application_wish_list (heavily outdated)

**Comment by trunkmaster** [comment]
Fine. Do you plan to implement something from that list?

**Comment by alexmyczko** [comment]
Guess I'm not able to. But I'll try to get some of your software built from your repo, using the (waiting for the new libobjc2 1.9/2.0 release), and the new gorm release, http://livecd.gnustep.org/ a 3.0 or so... Like your extra settings on Preferences and Workspace, and Login.

**Comment by trunkmaster** [comment]
Be aware that NEXTSPACE code was not tested on new libobjc and GNUstep. Also note that GNUstep libraries need changes to make Workspace's new window management features work. Anyway - good luck! And keep me informed on progress.

**Comment by alexmyczko** [comment]
I'll pass. waiting for you to update all the stuff to latest gnustep libs

**Comment by trunkmaster** [comment]
OK. Once I'll managed to complete my SoundKit implementation effort (at least for Sound Prererences needs)... ;)

**Comment by armm77** [comment]
Amazing work on your project, I would like to have time to contribute something.
